package.json

```diff
@@ -6,7 +6,6 @@
     "build": "npm run clean && babel -d ./dist ./src -s",
     "postinstall": "npm run build",
     "start": "node ./dist/index.js",
-    "heroku-prebuild": "echo $IOS_APN_KEY_BASE64 | base64 --decode > $IOS_APN_KEY",
```

Comment:
I don't think this is needed.

Reply:
Yes we need this. This creates the p8 file.

Reply:
@gcolotti can you explain to me why this is needed? I didn't see any examples of this online.

Reply:
Yes. Before heroku starts to build the application, it runs this prebuild command. It basically is a bash command that generates a file in the root of the project, with the IOS_APN_KEY name/path and IOS_APN_KEY_BASE64 content. If you check those envs in our server config, you will see they correspond to the p8 file we use